xcode/Subconscious/Shared/Components/Common/Profile/EditProfileSheet.swift

```diff
@@ -140,6 +140,12 @@ struct EditProfileSheet: View {
         )
     }
     
+    var bioCaption: String.LocalizationValue {
+        String.LocalizationValue(
```

Comment:
I suspect we probably want to use the `String(localized:)` constructor in-place, rather than passing this type around. https://developer.apple.com/documentation/swift/string/init(localized:)